services/QuillLMS/app/helpers/world_history_1750_to_present_activities_helper.rb

```diff
@@ -118,6 +118,13 @@ def the_world_in_1750_activities
         description: 'Students will read and write about the trade relationship between Britain and China during the late 18th century.',
         paired_oer_asset_name: 'Qing Dynasty',
         paired_oer_asset_link: 'https://www.oerproject.com/OER-Materials/OER-Media/HTML-Articles/1750/Unit1/Qing-Dynasty?&lsid=f3a3fb7e-5a52-491c-a611-51147a0e51a8'
+      },
+      {
+        activity_id: nil,
```

Comment:
It looks like Ari has an activity id for this now -- he's linked to the evidence activity in this spec, but this should be that evidence activity's `parent_id`. Once that's added, we should also remove ` - Coming Soon!` from the `display_name` below.